docs/modules/administration/pages/installation.adoc

```diff
@@ -58,6 +58,8 @@ Add all files of directory `ods-configuration` to git and commit the result. You
 
 "Preparing" also updates the sample files and checks if any params are present in the sample files but missing from the real file.
 
+WARNING: If you are using windows **Cygwin** or **WSL** remember to change ending line CRLF to LF before commit `ods-core.env`
```

Comment:
How about linking to https://docs.github.com/en/github/using-git/configuring-git-to-handle-line-endings? There's a "Windows" section which explains how to apply some global config to avoid issues.

Reply:
good idea 👍